design/FY2022/enhacement_to_resilience_metric.md

```diff
@@ -0,0 +1,211 @@
+
+## Overview ##
+
+### 1.1 1. Hours of Safety ###
+
+Hours of Safety is a framework developed by EPA and RMI to help understand how long a home can maintain thresholds of comfort and safety before reaching unsafe indoor temperature levels [1]. The concept attempts to define the duration of time that homes can be expected to provide safe temperatures when the power goes out based on building characteristics and energy efficiency levels (e.g., insulation, infiltration). This metric can be used to quantify the amount of time people are exposed to extremely hot or cold temperatures indoors. The information can be used to guide weatherization efforts and emergency response measures in considering the health and safety of vulnerable populations as extreme weather events increase in frequency. 
+
+Hours of Safety for cold or hot weather events should be defined by the longest duration (number of hours), starting from the beginning time of the risk period (e.g., the start time of a power outage), to not exceed temperature thresholds defined for a specific type of population (e.g., cold stress safety temperature for the healthy population as 60°F, or 16°C). To define the thresholds, we will use EnergyPlus’ existing thermal comfort model controlled primarily by the People input object, and add two fields in the *People* object, namely "*Cold Stress Temperature Threshold*" and "*Heat Stress Temperature Threshold*". To define the risk period, we will add a new object in EnergyPlus to declare the start and end time to evaluate the resilience metrics within those dates. 
+
+The default "*Cold Stress Temperature Thresholds*": 60°F (16°C) for the healthy population [1]. Other values can be used: 64°F (18°C) for the elderly [1], and 71°F (22°C) for nursing home residents [2]. The default "*Hot Stress Temperature Threshold*" is selected as 86°F (30°C) [2]. Users can modify these default thresholds with the "*Cold Stress Temperature Threshold*" and "*Heat Stress Temperature Threshold*" input fields in the *People* object. 
+
+We will report both the zone and building-level Hours of Safety. For the zone level, we will report the longest duration in a risk period a zone can maintain comfort and safety before reaching unsafe indoor temperature thresholds. For the building level, we’ll provide four summary statistics across all zones in the building: min, max, average, and sum.
```

Comment:
Please add to the documentation that during events with power outages that models should add strategies that are typically  used by people in these circumstances such as increased airflow from window usage during hot events and the use of non-electric heating (fireplaces, stoves, etc.) during cold events. A model that does not take these considerations into account would be misleading. Especially for the hot events, EnergyPlus will show a sealed room get very hot.

Reply:
Thanks, the "Approach" section is updated about documentation updates suggesting users consider human adaptation strategies during hot and cold events in their models.